### PR TITLE
Update CPPE

### PR DESCRIPTION
**Pull request by maxscheurer**
## Description
This PR updates the `cppe` interface code (including option handling) to the newest version `v0.2.1` (major performance improvement 🚀). 

## Todos
Notable points (developer or user-interest) that this PR has or will accomplish.
- [x] Updated interface, options
- [x] Add options for Thole damping

## Questions
- [ ] Question1

## Checklist
- [x] Tests added for any new features
- [x] [All or relevant fraction of full tests run](http://psicode.org/psi4manual/master/build_planning.html#how-to-run-a-subset-of-tests)

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

File: psi4/driver/procrouting/proc.py [hunk 1]
```diff
@@ -1518,7 +1518,7 @@ def scf_helper(name, post_scf=True, **kwargs):
         # PE needs information about molecule and basis set
         pol_embed_options = solvent.pol_embed.get_pe_options()
         core.print_out(""" Using potential file {} for Polarizable Embedding
-                       calculation.\n""".format(pol_embed_options.potfile))
+                       calculation.\n""".format(pol_embed_options["potfile"]))
```
Comment (robertodr) [inline psi4/driver/procrouting/proc.py:1521]: Would you mind changing this to an f-string?

Comment (maxscheurer) [reply psi4/driver/procrouting/proc.py:1521]: Done with pleasure 😄

**Comment by andysim** [comment]
Just out of interest, can the power or R used in the Thole damping exponential be changed?

**Comment by maxscheurer** [comment]
> Just out of interest, can the power or R used in the Thole damping exponential be changed?

I guess anything could in principle be customized... (probably not in the current version) 😄 Anything specific you have in mind?

**Comment by andysim** [comment]
Thanks for the reply.  I don't have any immediate needs, but it might be something for the future.  The AMOEBA force field uses R^3 decay in the exponent, but the CHARMM Drude force field uses R instead.  I'm interested in using CPPE as we move towards QM/MM for these, so it would be nice to be able to set that eventually.

**Comment by maxscheurer** [comment]
For Amoeba, we have the Python code to generate the interaction tensors, so that's in principle easy. It's not in there yet because I almost never use damping myself 😄 Let me know if you need anything (Slack, issue on [CPPE repo](https://github.com/maxscheurer/cppe)...)

**Pull request merged**